src/main/java/graphql/normalized/ExecutableNormalizedOperation.java

```diff
@@ -23,7 +23,7 @@
  * An operation consists of a list of {@link ExecutableNormalizedField}s in a parent child hierarchy
  */
 @PublicApi
-public class ExecutableNormalizedOperation {
+public class ExecutableNormalizedOperation implements ParsedNormalizedOperation {
```

Comment:
Ahh you are going for a common interface - again I dont think its "parsed" - this is not parsing - parsing has a very specific name in this code base - eg text -> AST elements

Reply:
Yes this bit I expected some discussion. Possibly I should rename the new normalized operation implementation classes to something else then use `NormalizedOperation` as the interface name, otherwise if there is a better interface name then we can change to that. Let me know what you think?

Reply:
For reference here is the follow up PR (WIP) to enable a config to switch between `ExecutableNormalizedOperation` and `NormalizedOperation`. #4047

Reply:
Ahh didnt see your replies when I made more naming suggestions

Reply:
> name storming
> 
> NormalizedOperation + ExecutableNormalizedOperation
> 
> What is the common name here?
> 
> NormalOperation
> GraphQlOperation
> GraphQlNormalizedOperation

Leaning towards `GraphQlNormalizedOperation`.

Reply:
@bbakerman and I had a chat and thinking more about it I am not 100% sure we really want an interface and two different implementations. 

Not only the naming makes it clear that this is very hard to distinguish ... maybe we really want to have just NormalizedField which is a combination of ENF and the current NF.

Reply:
I see. I will still need an interface internally to allow the switching between ENO and NO for experimental usage, but can hide that.

Re the instrumentation do we want a separate method for executable normalized operation and normalized document creation, or should I just create a class container that can hold either types?